src/cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBufferExtensions.cs

```diff
@@ -607,8 +618,21 @@ private static IntPtr GetPointerAtOffset(this IBuffer buffer, uint offset)
                 IntPtr buffPtr = buffer.As<IBufferByteAccess>().Buffer;
                 return new IntPtr((byte*)buffPtr + offset);
             }
+        }
+
+        private static Span<byte> GetSpanForCapacity(this IBuffer buffer, uint offset)
+        {
+            Debug.Assert(0 <= offset);
+            Debug.Assert(offset < buffer.Length);
+
+            unsafe
+            {
+                IntPtr buffPtr = buffer.As<IBufferByteAccess>().Buffer;

```

Comment:
Isn't `As<T>()` the non trim-friendly version? We should be passing the IID explicitly, no?

Reply:
I noticed that too, I didn't comment on this because I noticed `GetPointerAtOffset` right above does the same, so I will probably need to go through these buffer extensions to make sure they are using the right AOT safe functions.  But the IID parameter one is probably the right one to use.

Reply:
Do you wanna fix separately?

Reply:
If you want to fix this new instance here, thats fine.

Reply:
Discussed offline that we'll look at this separately.